pages/thc/[processRef]/manager-review.tsx

```diff
@@ -0,0 +1,269 @@
+    : "Loading...";
+
+  const isUnannouncedVisit = useDataValue(
+    Storage.ProcessContext,
+    "isUnannouncedVisit",
+    processRef,
+    (values) => (processRef ? values[processRef] : undefined)
+  );
+
+  const getManagerSummaryText = (
+    yesValue: string,
+    noValue: string,
+    databaseValue: { value: string; notes: Notes }
+  ): string => {
+    if (databaseValue.value === "yes") {
+      return yesValue;
+    } else {
+      return databaseValue.notes.length > 0
+        ? `${noValue}. Note: ${databaseValue.notes[0].value}`
+        : `${noValue}`;
+    }
+  };
+
+  const visitType = isUnannouncedVisit.result
+    ? getManagerSummaryText(
+        "Unannounced",
+        "Announced",
+        isUnannouncedVisit.result
+      )
+    : "Loading...";
+
+  const isVisitInside = useDataValue(
+    Storage.ProcessContext,
+    "isVisitInside",
+    processRef,
+    (values) => (processRef ? values[processRef] : undefined)
+  );
+
+  const visitTookPlace = isVisitInside.result
+    ? getManagerSummaryText("Inside", "Outside", isVisitInside.result)
+    : "Loading...";
+
+  const otherNotes = useDataValue(
+    Storage.ProcessContext,
+    "otherNotes",
+    processRef,
+    (values) => (processRef ? values[processRef] : [])
+  );
+
+  const otherNotesValues = otherNotes.result
+    ? otherNotes.result.map(({ value }) => value)
+    : [];
+
+  const signatures = useDataSet(
+    Storage.ResidentContext,
+    "signature",
+    tenantIdsPresentForCheckValue
+  );
+
+  const signatureValues = signatures.result ? signatures.result : {};
+
+  const allTenantNames = tenantsValue.map(({ fullName }) => fullName);
+
+  const tenantsPresent = tenantsValue.filter((tenant) =>
+    tenantIdsPresentForCheckValue.includes(tenant.id)
+  );
+
+  const presentTenantNames = tenantsPresent.map(({ fullName }) => fullName);
+
+  const outsidePropertyImages = useDataValue(
+    Storage.ProcessContext,
+    "property",
+    processRef,
+    (values) => (processRef ? values[processRef]?.outside : undefined)
+  );
+
+  const outsidePropertyImageThumbnails = outsidePropertyImages.result
+    ? outsidePropertyImages.result.images.map((image) => (
+        <Thumbnail
+          key={image}
+          src={image}
+          alt={"Thumbnail of an uploaded image"}
+        />
+      ))
+    : [];
+
+  const extraRows = [
+    { key: "Outside property", value: outsidePropertyImageThumbnails },
+    { key: "Completed by", value: officerFullName },
+    { key: "Date completed", value: submittedDateValue },
+    { key: "The visit was", value: visitType },
+    { key: "The visit took place", value: visitTookPlace },
+    { key: "Reason for visit", value: reasonForVisit },
+    {
+      key: "Present for check",
+      value:
+        presentTenantNames.length > 0
+          ? presentTenantNames.join(", ")
+          : "Loading...",
+    },
+  ].filter(({ value }) => value.length > 0);
+
+  return (
+    <MainLayout
+      title={PageTitles.Review}
```

Comment:
```suggestion
      title={PageTitles.ManagerReview}
```

Even if they're the same.